src/cache.rs

```diff
@@ -160,22 +160,49 @@ impl Cache {
         let platform = self.get_platform_dir();
 
         // Search for the page in the platform specific directory
-        if let Some(pf) = platform {
-            let path = platforms_dir.join(&pf).join(&page_filename);
-            if path.exists() && path.is_file() {
-                return Some(path);
+        let pf_path = if let Some(pf) = platform {
+            let p = platforms_dir.join(&pf).join(&page_filename);
+            if p.exists() && p.is_file() {
+                Some(p)
+            } else {
+                None
             }
-        }
+        } else {
+            None
+        };
 
         // If platform is not supported or if platform specific page does not exist,
         // look up the page in the "common" directory.
-        let path = platforms_dir.join("common").join(&page_filename);
+        let common_path = platforms_dir.join("common").join(&page_filename);
+        let common_path = if common_path.exists() && common_path.is_file() {
+            Some(common_path)
+        } else {
+            None
+        };
+
+        // Get custom pages directory
+        let custom_env = if let Ok(value) = env::var("TEALDEER_CUSTOM_PAGES_DIR") {
```

Comment:
I would prefer if this would be a configuration file option, and not an environment variable.

Maybe like this?

```toml
[directories]
custom_pages_dir = "/abs/path/to/dir"
```

We could also use a default value in the config directory as fallback, e.g. `CONFIGDIR/pages/`, if not set. That way, custom pages could be added without having to set up a config variable.